app/views/repositories/show.html.erb

```diff
@@ -5,13 +5,21 @@
     <strong>Image:</strong> <%= @repository.image %>
   </div>
   <% if @repository.tags.any? %>
-    <div class="list-group">
-      <% @repository.tags.sort.each do |tag| %>
-        <%= link_to tag_path(repo: @repository.name, tag: tag), class: "list-group-item" do %>
+    <ul class="list-group">
+      <% @repository.tags.sort.reverse.each do |tag| %>
+        <li class="list-group-item">
+        <%= link_to tag_path(repo: @repository.name, tag: tag) do %>
           <strong>Tag:</strong> <%= tag %>
         <% end %>
+        <% if Rails.configuration.x.public_registry_url %>
+          <div class="pull-right">
+            <span id="pull-command-<%= tag.object_id %>"><%= get_pull_command(@repository.name, tag)%></span>
```

Comment:
Looks like you forgot to change the method name here:
`get_pull_command` -> `pull_command`.